test/functional/feature_block.py

```diff
@@ -78,7 +78,7 @@ class FullBlockTest(BitcoinTestFramework):
     def set_test_params(self):
         self.num_nodes = 1
         self.setup_clean_chain = True
-        self.extra_args = [[]]
+        self.extra_args = [['-acceptnonstdtxn=1']]  # This is a consensus block test, we don't care about tx policy
```

Comment:
Most tests shouldn't care about tx policy (only policy-specific tests should), so this should be set by default in the test framework.

Reply:
Tests should not begin to fail just because the node policy changes (unless those tests are specifically covering policy code).